src/MFeatures.cpp

```diff
@@ -148,11 +152,88 @@ void MFeatures::setMSfmData(MSfMData* sfmData)
   Q_EMIT sfmDataChanged();
 }
 
+void MFeatures::loadGlobalInfo()
+{
+  // safety check
+  if (_mtracks == nullptr || _mtracks->status() != MTracks::Ready) {
+    setGlobalInfoStatus(None);
+    return;
+  }
+  if (_msfmData == nullptr || _msfmData->status() != MSfMData::Ready) {
+    setGlobalInfoStatus(None);
+    return;
+  }
+  // get all viewIds from sfmData
+  std::vector<aliceVision::IndexT> viewIds;
+  getAllViewIds(viewIds);
+  if (viewIds.empty()) {
+    setGlobalInfoStatus(None);
+    return;
+  }
+  
+  // load features from file in a seperate thread
+  qDebug("[QtAliceVision] Track global info: Load features from file in a seperate thread.");
+  setGlobalInfoStatus(Loading);
+  FeaturesIORunnable* ioRunnable = new FeaturesIORunnable(FeaturesIORunnable::IOParams(_featureFolder, viewIds, _describerTypes));
+  connect(ioRunnable, &FeaturesIORunnable::resultReady, this, &MFeatures::updateGlobalTrackInfo);
+  QThreadPool::globalInstance()->start(ioRunnable);
+}
+
+void MFeatures::updateGlobalTrackInfo(MViewFeaturesPerViewPerDesc* viewFeaturesPerViewPerDesc)
+{
+  if (!viewFeaturesPerViewPerDesc) {
+    setGlobalInfoStatus(None);
+    return;
+  }
+
+  bool updated = updateFromTracks(viewFeaturesPerViewPerDesc) && updateFromSfM(viewFeaturesPerViewPerDesc);
```

Comment:
`updateFromTracks` and `updateFromSfM` should not block the extrated features display.
Extracted features should be visible even if StructureFromMotion is not computed.